app/views/core/RootView.coffee

```diff
@@ -124,6 +124,9 @@ module.exports = class RootView extends CocoView
 
     if title = @getTitle() then title += ' | CodeCombat'
     else title = 'CodeCombat - Learn how to code by playing a game'
+    
+    if localStorage?.showViewNames
+      title = @constructor.name
```

Comment:
Can it be in `application.debug.showViewNames` so we can put more debugging stuff in there, and have it be clear its for debugging and not something the code should normally call ?